roles/keystone/tasks/k2k-sp.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: install libapache2-mod-shib2
+  apt: name=libapache2-mod-shib2 state=present
+
+- name: install shibboleth private key
+  template: src=etc/shibboleth/sp-key.pem dest=/etc/shibboleth/sp-key.pem
+            owner=_shibd group=_shibd mode=0600
+
+- name: install shibboleth public certificate
+  template: src=etc/shibboleth/sp-cert.pem dest=/etc/shibboleth/sp-cert.pem
+            owner=_shibd group=_shibd mode=0644
+
+- name: configure shibboleth identity attributes
+  template: src=etc/shibboleth/attribute-map.xml
+            dest=/etc/shibboleth/attribute-map.xml mode=0600
+            owner=_shibd group=_shibd
+
+- name: configure shibboleth settings
+  template: src=etc/shibboleth/shibboleth2.xml
+            dest=/etc/shibboleth/shibboleth2.xml mode=0644
+            owner=_shibd group=_shibd
+
```

Comment:
Or this call to start shibboleth should live near where we start keystone itself, so that the other handler flush is used. (and of course this task would be properly conditional)